index.d.ts

```diff
@@ -1,6 +1,7 @@
 import { IFlagsmith } from './types';
 declare const flagsmith: IFlagsmith;
 export default flagsmith;
+export * from './types';
```

Comment:
I'd love to understand the use of the wildcard export here. In a python context, whenever I see an `import *`, it's an immediate red flag, but maybe that's not the case here because `/types` explicitly defines what it wants to export I suppose?

Reply:
This'll basically mean we can import anything that was exported in types directly from flagsmith, i.e. it'll let us do 

```
import { IInitConfig, IState } from 'flagsmith';
```
rather than need to do
```
import { IInitConfig, IState } from 'flagsmith/types';
```

It's an improvement but not really part of the fix to the original issue.